KsApi/mutations/inputs/PaymentSourceDeleteInput.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public struct PaymentSourceDeleteInput: GraphMutationInput {
+  let paymentSourceId: String
```

Comment:
Not sure why the input for this is `paymentSourceId` and not just `id`. We could create a reusable input type like `IdOnlyInput` for this sort of thing if it were.

Reply:
That's a good idea. it's similar to the `EmptyInput` we have.

Reply:
So I think whenever we use `id` in graph it refers to the relay ID, but in the case of payment methods the `id` we care about is actually the Rosie id because I believe the type `CreditCard` doesn't conform to the `Node` interface. So its better that it's explicitly named `paymentSourceId` to avoid ambiguity about what the id really is.

Reply:
Ah right, thanks for explaining!